src/go/k8s/api/redpanda/v1alpha1/redpanda_conversion_test.go

```diff
@@ -56,17 +123,16 @@ func TestConvertTo(t *testing.T) {
 
 	for _, tc := range []struct {
 		name string
-		flag *v1alpha1.CloudStorageEnabledString
+		flag *v1alpha1.BoolString
 	}{
 		{"nil", nil},
-		{"enabled", ptr.To(v1alpha1.CloudStorageEnabledString("true"))},
-		{"disabled", ptr.To(v1alpha1.CloudStorageEnabledString("false"))},
-		{"random", ptr.To(v1alpha1.CloudStorageEnabledString(random))},
+		{"enabled", ptr.To(v1alpha1.BoolString("true"))},
+		{"disabled", ptr.To(v1alpha1.BoolString("false"))},
 	} {
-		t.Run(fmt.Sprintf("cloud-enabled-%s", tc.name), func(t *testing.T) {
+		t.Run(fmt.Sprintf("cloud-enabled-%v", tc.name), func(t *testing.T) {
```

Comment:
NIT: If Sprintf is changed to `%v` then tests could be changed and test case `name` could be removed.
```suggestion
		t.Run(fmt.Sprintf("cloud-enabled-%v", tc.flag), func(t *testing.T) {
```

Reply:
done.